Makefile

```diff
@@ -50,12 +50,33 @@ run-render-mode:
 
 down:
 	docker compose -f src/$(DOCKER_COMPOSE) down
+	docker container stop grafana mqtt-broker
```

Comment:
docker compose down will already stop the grafana and mqtt-broker services - please remove this unnecessary line.